src/Rule/UrlHandler.php

```diff
@@ -36,9 +36,9 @@ public function validate(mixed $value, object $rule, ValidationContext $context)
             return $result;
```

Comment:
The class description in the current state is a copy paste from corresponding rule. Needs to be adjusted. At least "this rule" part is incorrect and handler can be linked with rule.